README.md

```diff
@@ -1 +1,59 @@
 # sygaldry
+
+## Building Sygaldry
+```bash
+go build -o sygaldry
+```
+
+## Dependencies
+```bash
+go get -u -v github.com/smallfish/simpleyaml
+```
+
+## Using the CLI
+```bash
+sygaldry build -f mySygaldryYamlFile.yaml
+```
+
+## YAML
+
+### Formatting
+```yaml
```

Comment:
We may want to wait until we have some real runes before we document these

Reply:
We can do that. I just figured first pass this was stuff we had talked about.